environment-dev.yml

```diff
@@ -10,7 +10,7 @@ dependencies:
   - xeus>=5.0.0
   - xeus-zmq>=3.0,<4.0
   - nlohmann_json=3.11.3
-  - CppInterOp>=1.5.0
```

Comment:
We should drop the version and it should take the newest. It can work with 1.5 too.

Reply:
Correct. Its was being enforced since the beginning like that I think though shouldn't be needed.

Reply:
Done. Should be ready now